main/data/default_encryption_dialog.ui

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.22.2 -->
+<interface>
+  <requires lib="gtk+" version="3.6"/>
+  <object class="GtkDialog" id="dialog">
+    <property name="can_focus">True</property>
+    <property name="modal">True</property>
+    <property name="default_width">320</property>
+    <property name="default_height">260</property>
+    <child type="titlebar">
+      <placeholder/>
+    </child>
+    <child>
+      <object class="GtkBox">
+        <property name="can_focus">False</property>
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="GtkBox">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="orientation">vertical</property>
+            <child>
+              <object class="GtkLabel" id="default_encryption_warning_label">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="label" translatable="yes">No default end-to-end encryption method has been previously selected for this conversation. XMPP defines some XEP for end-to-end encryption that Dino supports.
+
+It is strongly recommended that one of the following end-to-end encryption methods below is selected before sending a message.
+
+TAKE INTO ACCOUNT UNENCRYPTED CONVERSATIONS COULD BE READ BY THIRD PARTIES.
+
+These settings can be later changed from the Settings menu.</property>
```

Comment:
The text is long, technical, contains redundancy and partly uses all-capital letters that are usually perceived as shouting. It is also slightly misleading by mixing global default encryption and  encryption for the given conversation.

This could potentially confuse or annoy users and doesn't match the simple interface of Dino.

```suggestion
                <property name="label" translatable="yes">You are opening a new conversation without having set end-to-end encryption by default.

It is strongly recommended to enable it to prevent your messages from being read by third parties.

Please select an option to start this conversation with. Choosing one of the encryption methods will also set it as default in the global settings.</property>
```

Reply:
I agree your suggestion sounds less aggressive, while still making the point clear. Thanks!